db/model/model.js

```diff
@@ -1,19 +1,39 @@
 const db = require("../../db/connection");
+// const articles = require("../data/test-data/articles");
 
 exports.getTopic = () => {
   return db.query("SELECT * FROM topics").then((topics) => {
-    console.log(topics);
     return topics;
   });
 };
 
 exports.getArticleById = (id) => {
-  return db
-    .query(`SELECT * FROM articles WHERE article_id = ${id}`)
-    .then((result) => {
-      if (result.rows.length === 0) {
-        return Promise.reject({ status: 400, msg: "bad request" });
-      }
-      return result;
-    });
+  let queryValues = [];
+  let queryStr = `SELECT * FROM articles`;
+  if (id) {
+    queryValues.push(id);
+    queryStr += ` WHERE article_id = $1`;
+  }
+  queryStr += " ORDER BY created_at";
+
+  return db.query(queryStr, queryValues).then(({ rows }) => {
+    if (rows.length === 0) {
```

Comment:
if `rows.length === 0` that means you didn't find anything. Is that a `bad request `or a `not found`?